### PR TITLE
Clinvar sv fixes

### LINKED ISSUE #3829: Minor ClinVar SV submission issues
The clinvar variant sub page for SVs has some leftover hard-to-see button texts eg
Screenshot 2023-02-24 at 16 56 25

When picking svs to submit it can be tricky to tell them apart based on bands only:
Screenshot 2023-02-24 at 16 58 08
Perhaps we could add a little more info, like a coord to the line?

Also, it would be nifty if that variant type dropdown was autofilled with the variant subtype, like DEL or DUP.

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #3829 
- Fixed visibility of SV type in ClinVar submission form
- Try to pre-populate the SV type select given the SV subtype

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. Test locally using a SV variant of the following types: "ins", "del", "dup", "inv", "bnd"

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [ ] tests executed by

### CODE REVIEW

**Comment by northwestwitch** [comment]
Tested locally with this variant:

image

Gets the following type:

image

File: scout/constants/clinvar.py [hunk 1]
```diff
@@ -142,6 +142,14 @@
     "Complex",
 ]
 
+SCOUT_CLINVAR_SV_TYPES_MAP = {
+    "ins": "Insertion",
+    "del": "Deletion",
+    "dup": "Duplication",
+    "inv": "Inversion",
+    "bnd": "Complex",
```
Comment (dnil) [inline scout/constants/clinvar.py:150]: 👍

File: scout/server/blueprints/clinvar/templates/multistep_add_variant.html [hunk 2]
```diff
@@ -136,7 +136,7 @@ <h2 class="fs-title">Variant Details</h2>
                       <div class="col-6">
                       <!-- Variant type -->
                         {{variant_data.var_form.var_type.label(class="fw-bold, text-dark")}}
-                        {{variant_data.var_form.var_type(class="form-control, bg-white")}}
+                        {{variant_data.var_form.var_type(class="form-control, btn-secondary")}}
```
Comment (dnil) [inline scout/server/blueprints/clinvar/templates/multistep_add_variant.html:139]: 💯

**Review by dnil — APPROVED** [review]
Nice, thank you!

**Pull request merged**